x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/data_types_definition.tsx

```diff
@@ -122,15 +122,43 @@ export const TYPE_DEFINITION: { [key in DataType]: DataTypeDefinition } = {
     documentation: {
       main: '/date.html',
     },
+    description: () => (
+      <p>
+        <FormattedMessage
+          id="xpack.idxMgmt.mappingsEditor.dataType.dateLongDescription"
+          defaultMessage='Date fields accept strings containing formatted dates, e.g. "2015-01-01" or "2015/01/01 12:10:30", a long number representing milliseconds-since-the-epoch, or an integer representing seconds-since-the-epoch. Multiple date formats can be specified. Dates with timezone information will be converted to UTC internally.'
```

Comment:
Date fields accept strings with formatted dates ("2015-01-01" or "2015/01/01 12:10:30"), long numbers representing milliseconds since the epoch, and integers representing seconds since the epoch. Multiple date formats are allowed.  Dates with timezones are converted to UTC.

CJ's suggestion for "You can specify multiple date formats" also works, if you prefer.

Reply:
Latest text:
<img width="725" alt="Screen Shot 2020-01-08 at 9 44 12 AM" src="https://user-images.githubusercontent.com/5226211/71987265-e06ccb00-31fb-11ea-9c3f-a04bd92f82e1.png">

Reply:
This description is a little long.  How about removing "2015-01-01"?